basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,60 @@
+package basiclibrary;
+
+public class Library {
+    public static int[] roll(int n) {
+        int[] rolls = new int[n];
+
+        for (int i=0; i < rolls.length; i++){
+            int ranRoll = (int)Math.ceil(Math.random() * 6);
+            rolls[i] = ranRoll;
+        }
+
+        return rolls;
+    }
+
+    public static boolean containsDuplicates(int[] arr) {
+        if (arr.length == 0) return false;
+
+        int count = 0;
+
+        while (count < arr.length - 1) {
```

Comment:
it's standard practice to use a `for` loop when you know how many time you'll be looping (in this case `arr.length - 1`)